homeassistant/components/axis/axis_base.py

```diff
@@ -8,6 +9,25 @@
 from .const import DOMAIN as AXIS_DOMAIN
 from .device import AxisNetworkDevice
 
+TOPIC_TO_NAME = {
```

Comment:
Maybe `TOPIC_TO_EVENT_TYPE` to match https://github.com/home-assistant/core/pull/85431/files#diff-fa7a68f92ddf284bd902b165d415c6a51851aa8c974fc54a604a185e06b06ca3R74?

Reply:
Absolutely